Mage.Tests/src/test/java/org/mage/test/cards/single/iko/ZilorthaStrengthIncarnateTest.java

```diff
@@ -0,0 +1,160 @@
+package org.mage.test.cards.single.iko;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.After;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * @author htrajan
+ */
+public class ZilorthaStrengthIncarnateTest extends CardTestPlayerBase {
+
+    @After
+    public void after() {
+        assertAllCommandsUsed();
+    }
+
+    @Test
+    public void testNotPresent_combatDamageResolvesLethalityAsNormal() {
+        addCard(Zone.BATTLEFIELD, playerA, "Savai Sabertooth");
+        addCard(Zone.BATTLEFIELD, playerB, "Drannith Healer");
+
+        attack(1, playerA, "Savai Sabertooth");
+        block(1, playerB, "Drannith Healer", "Savai Sabertooth");
+
+        execute();
+
+        assertGraveyardCount(playerA, "Savai Sabertooth", 1);
+        assertGraveyardCount(playerB, "Drannith Healer", 1);
+    }
+
+    @Test
+    public void testPresent_combatDamageResolvesLethalityUsingPower() {
+        addCard(Zone.BATTLEFIELD, playerA, "Zilortha, Strength Incarnate");
+        addCard(Zone.BATTLEFIELD, playerA, "Savai Sabertooth");
+        addCard(Zone.BATTLEFIELD, playerB, "Drannith Healer");
+
+        attack(1, playerA, "Savai Sabertooth");
+        block(1, playerB, "Drannith Healer", "Savai Sabertooth");
+
+        execute();
```

Comment:
Use `setStrictChooseMode` and `assertAllCommandsUsed` for new tests. Example:
```
setStrictChooseMode(true);
setStopAt(3, PhaseStep.END_COMBAT);
execute();
assertAllCommandsUsed();
```

Reply:
fixed